src/Optimization/UI/AxisymmetricOptimization.jl

```diff
@@ -0,0 +1,349 @@
+# MIT license
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# See LICENSE in the project root for full license information.
+
+module AxisymmetricOptimization
+
+using OpticSim, OpticSim.Geometry, OpticSim.Emitters
+using CSV, DataFrames
+using Ipopt
+using JuMP
+import DataStructures
+
+global global_model = nothing
+global global_ax = nothing
+global global_df = nothing
+global global_counter = 0
+global solution_history = []
+global statistics = Dict()
+global options = Dict()
```

Comment:
Do you think we can put these in a struct that defines an Axisymmetric Optimization problem? I generally feel that global variables are inflexible - this would be more modular, making it easier to implement features such as saving/loading sessions in the future.